ARM-wvd-templates/HCI/CreateHciHostpoolTemplate.json

```diff
@@ -0,0 +1,490 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "hostPoolName": {
+        "type": "string",
+        "metadata": {
+          "description": "The name of the Hostpool to be created."
```

Comment:
change to-
Host pools are a collection of one or more identical virtual machines (VMs), also known as "session hosts," within Azure Virtual Desktop environments.